deploy/README.md

```diff
@@ -161,16 +162,57 @@ kubectl -n sumologic create secret generic sumologic \
   --from-literal=endpoint-events=$ENDPOINT_EVENTS
 ```
 
-Apply `fluentd-sumologic.yaml` manifest with following command:
+Get the `fluentd-sumologic.yaml` manifest with following command:
 
 ```sh
 curl https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/master/deploy/kubernetes/fluentd-sumologic.yaml.tmpl | \
-sed 's/\$NAMESPACE'"/sumologic/g" | \
+sed 's/\$NAMESPACE'"/sumologic/g"
+```
+
+You can customize the provided YAML file if needed, including configuring fields for [events](#events). Once done run the following command to apply the `fluentd-sumologic.yaml` manifest.
+
+```sh
 kubectl -n sumologic apply -f -
 ```
 
 The manifest will create the Kubernetes resources required by Fluentd.
 
+### Events
+
+To customize fields for events edit the provided (`fluentd-sumologic.yaml`) file by specifying the fields in the `events` plugin `@type` parameter. The following example has specified the `type_selector`, `deploy_namespace`, and `configmap_update_interval_seconds` fields,
+
+```json
+<source>
+    @type events
+    deploy_namespace $NAMESPACE
+    type_selector ["ADDED", "MODIFIED", "DELETED"]
+    configmap_update_interval_seconds 5
+</source>
+```
+
+#### Configurable fields for events
```

Comment:
Do we need to put these in our main `Deploy` README, or can these plugin-specific attributes be documented in the plugin's README?  I think that's more scalable than adding every attribute for every plugin to our `Deploy` README.  What do you guys think?

Reply:
Yeah I like that idea

Reply:
How do we do this? A link?

Reply:
Yes, in this same PR you can move the table of parameters to the `fluent-plugin-events` README (currently it just says `TODO: write description for you plugin.`), and then link to that plugin's documentation with a relative link.